src/devices/video/huc6270.h

```diff
@@ -21,15 +286,24 @@ class huc6270_device : public device_t
 	void set_vram_size(uint32_t vram_size) { m_vram_size = vram_size; }
 	auto irq() { return m_irq_changed_cb.bind(); }
 
+	// 8 bit I/O
 	u8 read(offs_t offset);
 	void write(offs_t offset, u8 data);
+	// TODO: 16 bit I/O used?
+	/*
+	u16 read16(offs_t offset);
+	void write16(offs_t offset, u16 data);
+	*/
```

Comment:
Internally HuC6270 **is** 16-bit, it's just the interface with PCE HuC6280 that is 8-bit, at least with normal opcodes (no idea with ST0/ST1/ST2 opcodes). And even then, PCFX V810 can do definitely go up to 32, so the whole point is that register access and internal state should be isolated into a single `io_space` that sets up `mem_mask` accordingly and nuke the dual switch-case.

TL;DR both notes are superfluous, plus your commented signatures are wrong (missing mem_mask)